edx_exams/apps/router/interop.py

```diff
@@ -0,0 +1,25 @@
+"""
+Module for writing data to edx-proctoring
+"""
+from posixpath import join as urljoin
+
+from django.conf import settings
+from requests.exceptions import HTTPError
+
+from edx_exams.apps.core.rest_utils import get_client, make_request
+
+LMS_REGISTER_PROCTORED_EXAMS_API_TPL = 'api/edx_proctoring/v1/proctored_exam/exam_registration/course_id/{}'
+
+
+def register_exams(course_id, exam_list):
+    """
+    Register a list of course exams with the legacy proctoring service
+    """
+    path = LMS_REGISTER_PROCTORED_EXAMS_API_TPL.format(course_id)
```

Comment:
Suggestion, refactor the URL construction into it's own function so it can be reused later. Make the URL path template a bit more generic would be nice too.